web.esphome.io/src/dashboard/unsupported-card.ts

```diff
@@ -23,6 +23,9 @@ class EWUnsupportedCard extends LitElement {
       esphome-card {
         --status-color: var(--alert-error-color);
       }
+      .card-actions mwc-button {
+        --mdc-theme-primary: var(--card-text-color);
```

Comment:
It looks wrong to set a theme color to a text color variable.

Reply:
This is what is currently done to change the color:
https://github.com/esphome/dashboard/blob/270f7e462bcf9e016f1169bca40548e091586ae8/web.esphome.io/src/dashboard/esp-connect-card.ts#L81-L83

In the docs it says "Color of a flat or outlined button. Background color of a raised button."
https://www.npmjs.com/package/@material/mwc-button#user-content-global-custom-properties